requirements.txt

```diff
@@ -1,2 +1,2 @@
 requests>=2.7.0
-pyrestcli>=0.6.9
+pyrestcli==0.6.11
```

Comment:
Should we let this be `>=0.6.11` in there are other bug fixes on pyrestcli?

Reply:
Yes, there is a bug in `pyrestcli` 0.6.12 or at least it is falling for us (it is looking for an unexpected property)

Reply:
👍